docs/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -26,6 +26,12 @@ description: "Learn how to enable Session Replay in your app if it is not alread
 
 By default, our Session Replay SDK masks all DOM text content, images, and user input, giving you heightened confidence that no sensitive data will leave the browser. To learn more, see <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
 
+<PlatformSection supported={["javascript.angular"]}>
+  <Alert level="danger">
+    Angular's <ExternalLink href="https://angular.dev/api/core/ChangeDetectionStrategy#Default">default "Change Detection" strategy</ExternalLink> monkeypatches browser globals and can break or cause performance regressions to your application when using Session Replay. We recommend you use the `OnPush` strategy when using Angular.
+  </Alert>
+</PlatformSection>
+
```

Comment:
```suggestion
<PlatformSection supported={["javascript.angular"]}>
  <Alert level="danger">
    Angular's <ExternalLink href="https://angular.dev/api/core/ChangeDetectionStrategy#Default">default "Change Detection" strategy</ExternalLink> monkeypatches browser globals and can break or cause performance regressions in your application when using Session Replay. To avoid this, we recommend you use the `OnPush` strategy when using Angular.
  </Alert>
</PlatformSection>

```